morphir/runtime/src/org/finos/morphir/runtime/quick/testing/TestTree.scala

```diff
@@ -251,9 +251,42 @@ private[runtime] object TestSet {
     TestSummary(
       toReport(testSet),
       testSet.modules.map(module => (module.pkgName, module.modName) -> ModuleTests.getCounts(module)).toMap,
-      coverageInfo
+      coverageInfo,
+      extractTestResults(testSet.modules)
     )
 
+  def extractTestResults(testSet: List[ModuleTests[SingleTestResult]])
```

Comment:
I don't think we should be exposing `ModuleTests` - that's really an implementation detail of how the tests are generated.